jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaPackager.kt

```diff
@@ -86,14 +79,22 @@ class PythonLambdaPackager : LambdaPackager {
         return future
     }
 
-    private fun addFolder(contentRoot: VirtualFile, excludedRoots: MutableSet<VirtualFile>, mappings: MutableMap<String, String>, zip: ZipOutputStream) {
-        contentRoot.walkFiles(excludedRoots) { file ->
-            VfsUtilCore.getRelativeLocation(file, contentRoot)?.let { relativeLocation ->
+    private fun addFolder(folderRoot: VirtualFile, excludedRoots: Set<VirtualFile>, mappings: MutableMap<String, String>, zip: ZipOutputStream) {
+        folderRoot.walkFiles(excludedRoots) { file ->
+            if (FILE_NAME_BLACKLIST.contains(file.name)) {
+                return@walkFiles
+            }
+
+            VfsUtilCore.getRelativeLocation(file, folderRoot)?.let { relativeLocation ->
                 mappings[file.path] = relativeLocation
                 file.inputStream.use { fileContents ->
                     zip.putNextEntry(relativeLocation, fileContents)
                 }
             }
         }
     }
+
+    private companion object {
+        val FILE_NAME_BLACKLIST = setOf(".DS_Store")
```

Comment:
Why do we remove the `thumbs.db` file?

Reply:
Seemed on that I found no reference to thumbs.db in IJ codebase so wasn't sure it was actually needed